lib/bills.js

```diff
@@ -9,15 +9,27 @@ define('bills', function (require) {
 			return this;
 		},
 		urlWithEntity: function() {
-			return Config.baseUrl+"/accounts";
+			return Config.baseUrl+'/bills/';
+		},
+		urlWithAccountEntity: function() {
+			return Config.baseUrl+'/accounts/';
+		},
+		urlWithCustomerEntity: function() {
+			return Config.baseUrl+'/customers/';
 		},
 		apiKey: function() {
 			return Config.apiKey;
 		},
-		getAllBillsByAccountId: function(accID) {
+		/**
+		  # @Method: getAllByAccountId
+		  # @Brief: Get all bills for a specific account
+		  # @Parameters: accountId
+		  # @Returns an array of hashes containing the bills.
```

Comment:
same here.  Anywhere there are hashes referenced we should remove them and update with the appropriate response value.